### DISCUSSION TITLE
Predefined document paths

### DISCUSSION THREAD

**Discussion by zulhfreelancer** [open-webui/open-webui#737] [Q&A] [answered]
From project's README, I see this:

> You can load documents directly into the chat or add files to your document library, effortlessly accessing them using # command in the prompt.

Is it possible to start ollama/ollama-web with predefined paths to files/directories that contain documents we want to chat with?

**Comment by justinh-rahb** [comment, marked as answer]
Not a feature currently, but being able to specify a directory (and mount it as a volume in Docker) would make an excellent PR 👍

**Reply by zulhfreelancer** [reply]
Thank you for the clarification 👍🏼

**Comment by DocMAX** [comment]
Wish to see that too

**Comment by jannikstdl** [comment]
This feature is now merged by our coding machine #778

**Reply by zulhfreelancer** [reply]
Awesome, thanks!
